MapboxDirections/MBLaneIndication.h

```diff
@@ -0,0 +1,29 @@
+
+typedef NS_OPTIONS(NSUInteger, MBLaneIndication) {
+    // Indicates a sharp turn to the right.
+    SharpRight = (1 << 4),
```

Comment:
Prefix each of these values with `MBLaneIndication`. In Swift, `MBLaneIndication` will be removed automatically.